### PR TITLE
Fix coding style issues #899

### PR DESCRIPTION
**Pull request by jongfeli**
Fixed some minor stuff and updated [Code-violations](https://github.com/jongfeli/Code-violations-SemanticMediaWiki/blob/master/all).

### CODE REVIEW

**Comment by mwjames** [comment]
Changes caused a
```
PHP Parse error:  syntax error, unexpected 'die' (T_EXIT) in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/SemanticMediaWiki.settings.php on line 25
PHP Stack trace:
PHP   1. {main}() /home/travis/build/SemanticMediaWiki/mw/maintenance/update.php:0
PHP   2. require_once() /home/travis/build/SemanticMediaWiki/mw/maintenance/update.php:194
PHP   3. require() /home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:90
PHP   4. require_once() /home/travis/build/SemanticMediaWiki/mw/LocalSettings.php:129
Parse error: syntax error, unexpected 'die' (T_EXIT) in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/SemanticMediaWiki.settings.php on line 25

```

File: SemanticMediaWiki.settings.php [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+*
+*/
 
 /**
- * @file
- * @ingroup SMW
- */
+* @file
+* @ingroup SMW
+*/
```
Comment (JeroenDeDauw) [inline SemanticMediaWiki.settings.php]: It was good before

File: SemanticMediaWiki.php [hunk 2]
```diff
@@ -117,13 +117,13 @@ class_alias( 'SMW\ParameterProcessorFactory', 'SMW\ParameterFormatterFactory' );
...
 // Load global constants
-require_once( __DIR__ . '/includes/Defines.php' );
+require_once ( __DIR__ . '/includes/Defines.php' );
 
 // Temporary measure to ease Composer/MW 1.22 migration
 require_once __DIR__ . '/includes/NamespaceManager.php';
 
 // Load global functions
-require_once( __DIR__ . '/includes/GlobalFunctions.php' );
+require_once ( __DIR__ . '/includes/GlobalFunctions.php' );
```
Comment (JeroenDeDauw) [inline SemanticMediaWiki.php]: This seems odd. The old style is better IMO. Even better is the following, which probably also makes the CS thing shut up:

```php
require_once __DIR__ . '/includes/GlobalFunctions.php'
```

**Comment by JeroenDeDauw** [comment]
@mwjames how do you figure that? Not seeing this in the CI. And did not spot anything in the PR that would cause this

**Comment by mwjames** [comment]
> how do you figure that? Not seeing this in the CI. And did not spot anything in the PR that would cause this

https://travis-ci.org/SemanticMediaWiki/SemanticMediaWiki/builds/80108061

**Comment by JeroenDeDauw** [comment]
Still not seeing it. IDE also being OK with the file. New PR at #1150

File: SemanticMediaWiki.settings.php (hunk 1 above)
Comment (jongfeli) [inline SemanticMediaWiki.settings.php]: OK, this means I will leave comments alone in the future :)

**Pull request closed without merging**